### DISCUSSION TITLE
install.ps1: A positional parameter cannot be found that accepts argument 'System.Collections.Hashtable'

### DISCUSSION THREAD

**Discussion by go-xoxo** [ScoopInstaller/Scoop#5895] [Help and Questions]
#### Current Behavior
Running installers that make use of line 1002 of the scoop install script install.ps1 
function env_set($manifest, $dir, $global, $arch) {
    $env_set = arch_specific 'env_set' $manifest $arch
    if ($env_set) {
        $env_set | Get-Member -Member NoteProperty | ForEach-Object {
            $name = $_.name;
-->            $val = format $env_set.$($_.name) @{ "dir" = $dir }
            env $name $global $val
            Set-Content env:\$name $val
        }
    }
}
I get the error A positional parameter cannot be found that accepts argument "System.Collections.Hashtable".The installer does not stop though and says "successfully installed"

#### Expected Behavior
no error messages during install of packages

### System details

**Windows version: 11
**OS architecture: 64bit

**PowerShell version: 7.4.0
**Additional software: none

#### Scoop Configuration
```json
{
  "last_update": "2024-01-01T13:17:37.2641227+01:00",
  "aria2-warning-enabled": false,
  "scoop_branch": "master",
  "scoop_repo": "https://github.com/ScoopInstaller/Scoop"
}

```

**Comment by rashil2000** [comment]
Which app are you installing?
